Geometry/HcalCommonData/interface/HcalDDDSimConstants.h

```diff
@@ -40,7 +40,7 @@ class HcalDDDSimConstants {
 					  const int zside) const;
   int                       getDepthEta16M(const int det) const;
   int                       getDepthEta29(const int phi, int zside, int i) const;
-  int                       getDepthEta29M(const int i) const;
+  int                       getDepthEta29M(const int i, const bool flag) const;
```

Comment:
Hi @bsunanda - here and in a few other places in this PR - could you either document what "flag" means or rather pick a more self documenting name?